test/support/helpers/charge-category.helper.js

```diff
@@ -74,11 +74,27 @@ function defaults (data = {}) {
   }
 }
 
+/**
+ * Generate Charge reference - e.g "4.6.42"
+ *
+ * A charge reference is built in three parts
+ *
+ * firstPart why - 4 (does it have a know range ? )
```

Comment:
Try and get answer for the why questions.

Reply:
Charge Categories are a reference record, not transactional.

Rather than tweak the generate, these should be moved to being a seed and the helper updated to just provide a `select()` method.

On why, because someone in policy liked the numbers (and that's all you'll ever know!)

Reply:
I will close this PR.